crates/bevy_asset/src/path.rs

```diff
@@ -48,9 +49,9 @@ impl<'a> Debug for AssetPath<'a> {
 
 impl<'a> Display for AssetPath<'a> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", self.path.display())?;
+        write!(f, "{:?}", &*self.path)?;
         if let Some(label) = &self.label {
-            write!(f, "#{label}")?;
+            write!(f, "#{}", &**label)?;
```

Comment:
Implement `Display` on `CowArc` should be better than manual dereferencing

Reply:
Agreed. `Path::display` might Alwine nicer than using the debug formatted in a `Display` impl :)

Reply:
Good call!